containers/payments/subscription/SubscriptionSection.js

```diff
@@ -14,42 +14,37 @@ import {
     useUser,
     useAddresses
 } from 'react-components';
-import { CYCLE, PLAN_NAMES, COUPON_CODES, BLACK_FRIDAY } from 'proton-shared/lib/constants';
+import { PLAN_NAMES } from 'proton-shared/lib/constants';
 import humanSize from 'proton-shared/lib/helpers/humanSize';
-import { isBundleEligible } from 'proton-shared/lib/helpers/subscription';
 
 import { formatPlans, toPlanNames } from './helpers';
-import DiscountBadge from '../DiscountBadge';
 import SubscriptionModal from './SubscriptionModal';
 
-const { MONTHLY, YEARLY, TWO_YEARS } = CYCLE;
-const { BUNDLE } = COUPON_CODES;
+const AddonRow = ({ label = '', used, max, format = (v) => v }) => {
```

Comment:
We have an `identity` function in `proton-shared/lib/function`. Not sure when we want to use it, but here it would be possible